plugin/pathvirt/pathvirt.cpp

```diff
@@ -0,0 +1,250 @@
+/* NOTE:  if you just want to insert your own code at the time of checkpoint
+ *  and restart, there are two simpler additional mechanisms:
+ *  dmtcpaware, and the MTCP special hook functions:
+ *    mtcpHookPreCheckpoint, mtcpHookPostCheckpoint, mtcpHookRestart
+ */
```

Comment:
@mossberg, dmtcpaware and mtcpHookPreCheckpoint are obsolete.  Here is some better wording for `*  and restart,  ...`:
```
 * and restart, the standard DMTCP plugins are available, using the events:
 *  DMTCP_EVENT_WRITE_CKPT and DMTCP_EVENT_RESUME and DMTCP_EVENT_RESTART.
 */